java/release_notes.txt

```diff
@@ -1,3 +1,14 @@
+May 03, 2016: libphonenumber-7.3.2
+Metadata changes:
+ - Updated phone metadata for region code(s):
+   AU, BR, ET, IN, KH, ML, NO, SB, TJ, US
+ - Updated short number metadata for region code(s): CA, ML, US
+ - New geocoding data for country calling code(s):
```

Comment:
No, but it's not about the binary file, it's about the text file, and that
was already there (not new). It just didn't generate the binary file before
because they were identical.

I think with the 1xxx we have oscillated back and forth.

2016-05-04 14:20 GMT+02:00 Keghani Kouzoujian <notifications@github.com>:

> In java/release_notes.txt
> <https://github.com/googlei18n/libphonenumber/pull/1102#discussion_r62028953>
> :
>
> > @@ -1,3 +1,14 @@
> > +May 03, 2016: libphonenumber-7.3.2
> > +Metadata changes:
> > + - Updated phone metadata for region code(s):
> > +   AU, BR, ET, IN, KH, ML, NO, SB, TJ, US
> > + - Updated short number metadata for region code(s): CA, ML, US
> > + - New geocoding data for country calling code(s):
>
> @lararennie <https://github.com/lararennie> Can we change this back to
> what was automatically generated?
>
> 55_pt was actually new this time (it was removed in a recent release):
> https://github.com/googlei18n/libphonenumber/commits/master/java/geocoder/src/com/google/i18n/phonenumbers/geocoding/data/55_pt
>
> And we've always included the entire 1xxx numbers according to the file
> names - see libphonenumber-7.1.1 release notes for example.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/googlei18n/libphonenumber/pull/1102/files/a3691f42228b7603a4014c392a95ecb0b7f452ba#r62028953>
>